### DISCUSSION TITLE
admob.BannerAd.config() for Android?

### DISCUSSION THREAD

**Discussion by lee-borlace** [admob-plus/admob-plus#335] [Ideas]
Hi there!

Building an app in Ionic with the Cordova plugin. I notice from looking at the code that admob.BannerAd.config() (https://admob-plus.github.io/docs/cordova/ads/banner#global-settings) only applies to iOS not Android.

Is there a reason for this? I'd really like to set the background colour of my banner ad to match the app's background. I'm assuming the background colour is the solid colour (black by default) between the edges of the ad and the sides of the screen.

I tried setting the offset, thinking it might hover over my main content, but the black bars followed it.

I'd like to do a PR to enable config() for Android (or even better have someone else implement it!) but just thought I'd check whether there's a reason for it not being there currently?

Cheers

Lee

**Comment by ratson** [comment]
@leeroy79 The API was added to address the iOS status bar issue, which Android does not have.
Feel free to send PR to implement it for Android.

**Reply by lee-borlace** [reply]
I see, doesn't sound like it's related to what I was talking about then. Would it be worth updating the docco to mention that it's a fix for a specific issue and only for iOS? I spent a bit of time trying to get config() to work in Android before I discovered it wasn't what I thought, and wasn't applicable anyway.

So there probably isn't any functionality currently to change the colour of the black background between the banner and the edges of the screen when the banner doesn't take the full width?
